src/enzo/Grid_SchrodingerSolver.C

```diff
@@ -170,8 +170,9 @@ int grid::SchrodingerSolver( int nhy )
   }
 
   // calculate hbar/m
+  float afloat = float(a);
+  float hmcoef = 5.9157166856e27*TimeUnits/pow(LengthUnits/afloat,2)/FDMMass;
```

Comment:
Two points: 

(1) Is it useful to ensure that this is always double, rather than float?  If people compile with 32-bit precision, it's likely that some combination of TimeUnits and LengthUnits, coupled with the ~6e27 fixed coefficient, may cause an overflow or underflow.

(2) Please use the POW() macro.